component-server-parent/component-server/src/dev/webapp/component.html

```diff
@@ -5,7 +5,7 @@ <h1>Sample</h1>
 <div id="metawidget">
 </div>
 <script src="https://unpkg.com/axios/dist/axios.min.js"></script>
-<script src="http://metawidget.org/js/4.2/metawidget-core.min.js"></script>
+<script src="https://metawidget.org/js/4.2/metawidget-core.min.js"></script>
```

Comment:
Should we set the sourceforge url since there is a 302:
```
$ curl https://metawidget.org/js/4.2/metawidget-core.min.js
<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>302 Found</title>
</head><body>
<h1>Found</h1>
<p>The document has moved <a href="https://metawidget.sourceforge.net/js/4.2/metawidget-core.min.js">here</a>.</p>
<hr>
<address>Apache/2.4.59 (Debian) Server at metawidget.org Port 443</address>
</body></html>
```

Reply:
Yep, but it fools security scanners apparently.